### PR TITLE
Don't ask personalisation questions which have already been answered

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: src/pages/PersonalisationWizardPage.js [hunk 1]
```diff
@@ -39,9 +39,10 @@ class PersonalisationWizardPage extends BasePersonalisationPage {
     }
 
     nextSubPage(): ?ReactClass {
-        const nextSubpageIdx = this.currentComponentIdx + 1;
-
-        return this.personalisationComponents[nextSubpageIdx];
+        return this.personalisationComponents.find((component, idx) =>
+            (idx > this.currentComponentIdx) &&
+            (!component.getSearch({}))
+        );
```
Comment (danni) [inline src/pages/PersonalisationWizardPage.js:45]: Overly complicated.

Just use `for component of ... { if (foo) return bar; }`

Comment (DanielHeath) [inline src/pages/PersonalisationWizardPage.js:45]: That sounds more complicated rather than less

Comment (DanielHeath) [inline src/pages/PersonalisationWizardPage.js:45]: `find a thing matching this` vs `loop through the things and return if one matches this`

**Comment by DanielHeath** [comment]
https://trello.com/c/8yFtsizm/256-ui-save-personalisation-details-in-a-cookie-small

File: src/pages/PersonalisationWizardPage.js (hunk 1 above)
Comment (DanielHeath) [inline src/pages/PersonalisationWizardPage.js:45]: As discussed - will not work as we need to check the index (which isn't available in a comprehension).

Comment (DanielHeath) [inline src/pages/PersonalisationWizardPage.js:45]: @danni anything else?

**Pull request merged**